lib/ZwaveClient.js

```diff
@@ -1186,6 +1186,8 @@ function updateValue (zwaveNode, args) {
           self.emit('valueChanged', valueId, node, false)
         }, 1000)
       }
+    } else {
+      addValue.call(this, zwaveNode, args)
```

Comment:
Wouldn't you want to apply the timeout logic here too?

Reply:
What? This just add the value to values so it will also be displayed on UI

Reply:
Yeah, but in valueChanged, you reset the value to undefined after one second. I don't see this happening here.

Reply:
right, fixed